package.json

```diff
@@ -181,7 +181,7 @@
     "**/@babel/preset-react": "^7.10.1",
     "**/@babel/preset-typescript": "^7.10.4",
     "**/@babel/runtime": "^7.10.2",
-    "**/@babel/types": "^7.10.2",
+    "**/@babel/types": "7.12.6",
```

Comment:
`@babel/types@7.12.7` introduces `typesVersions` which has no effect. `typescript-to-proptypes` is still using the version for typescript ^3.7 even though it uses 3.5. 

Needs investigation upstream at some point.